recipe/meta.yaml

```diff
@@ -84,3 +70,4 @@ extra:
     - bam241
     - katyhuff
     - FlanFlanagan
+    - stompsjo
```

Comment:
FYI: Since I have been working on implementing this 1.5.5 version, I would like to be added to the list of maintainers for this conda package.